pkg/local_object_storage/pilorama/boltdb.go

```diff
@@ -557,6 +557,31 @@ func (t *boltForest) TreeGetChildren(cid cidSDK.ID, treeID string, nodeID Node)
 	return children, err
 }
 
+// TreeGetTrees implements the Forest interface.
+func (t *boltForest) TreeGetTrees(cid cidSDK.ID) ([]string, error) {
```

Comment:
`TreeListAll`?

Reply:
renamed: `ListTrees` in RPC and `TreeList` in the `Forest` intreface